docs/server/configuration.md

```diff
@@ -26,23 +26,55 @@ Temporal Server configuration is found in `development.yaml` and may contain the
 The `global` section contains process-wide configuration. See below for a minimal configuration (optional parameters are commented out.)
 
 ```yaml
-#global:
-#membership:
-#maxJoinDuration: 30s
-#broadcastAddress: "127.0.0.1"
-#pprof:
-#port: 7936
-#tls:
-#... <see below>
+global:
+  membership:
+    broadcastAddress: "127.0.0.1"
+  metrics:
+    prometheus:
+      framework: "opentelemetry"
+      listenAddress: "127.0.0.1:8000"
 ```
 
 ### membership
 
 The `membership` section controls the following membership layer parameters:
 
-- `maxJoinDuration` - The amount of time the service will attempt to join the gossip layer before failing.
-- `broadcastAddress` - Used as the address that is communicated to remote nodes to connect on.
-  - This is generally used when BindOnIP would be the same across several nodes (ie: 0.0.0.0) and for nat traversal scenarios. `net.ParseIP` controls the supported syntax. Note: Only IPV4 is supported.
+- `maxJoinDuration` - The amount of time the service will attempt to join the gossip layer before failing. Default is 10s.
+- `broadcastAddress` - Used by gossip protocal to communicate with other nodes in the same cluster for membership info.
+  - Use IP address that is reachable by other nodes in the same cluster. Check `net.ParseIP` for supported syntax, only IPv4 is supported.
```

Comment:
Need some help on this section.

Reply:
This is the reachable address for ringpop membership, which translates to the IP address to bind to for service communication.

This generally 127.0.0.1 or 0.0.0.0 for most scenarios.

In some complex setups this would be the IP address reachable by other members needing to communicate with this node.